.github/workflows/build.yml

```diff
@@ -23,45 +24,52 @@ jobs:
       with:
         dotnet-version: 6.0.x
 
-    - name: Restore dependencies
-      run: dotnet restore
-
-    - name: Main Build
-      run: dotnet build --configuration $config --no-restore -p:FileVersionRevision=$GITHUB_RUN_NUMBER
-
-    - name: Docker Build - Chat Bot Sample
-      run: docker build -f samples/chat/csharp-inproc/Dockerfile -t openai-func-sample-csharp-inproc .
+    - name: Docker Build - Chat Bot Sample Out of proc
+      run: docker build -f samples/chat/csharp-ooproc/Dockerfile -t openai-func-sample-csharp-ooproc .
 
     - name: Docker Run - Azurite
       run: docker run -d -p 10000:10000 -p 10001:10001 -p 10002:10002 mcr.microsoft.com/azure-storage/azurite
 
-    - name: Docker Run - Chat Bot Sample
+    - name: Docker Build - Chat Bot Sample Out of proc
+      run: docker build -f samples/chat/csharp-ooproc/Dockerfile -t openai-func-sample-csharp-ooproc .
```

Comment:
Not sure if I'm reading this diff correctly, but it looks like we have two duplicate `docker build` steps?

Reply:
Thanks for catching that! Removed second docker build step